File: Level1/소수 찾기.cpp
#include <string>
#include <vector>
bool prime[1000001] = {false,};
using namespace std;

int solution(int n) {
    int answer = 0;
    for(int i = 2; i <= n; i++)
    {
        if(prime[i] == false)
           answer++;
        for(int j = i + i; j <= n; j += i)
            prime[j] = true;
    }
    return answer;
}

File: Level1/소수 찾기.js
function solution(n) {
    let answer = 0;
    let prime = Array.from({length: n + 1}, () => false);
    
    for(let i = 2; i <= n; i++){
        
        if(!prime[i]){
            answer++;
            for(let j = i + i; j <= n; j += i){
                prime[j] = true;
            }
        }
    }
    
    return answer;
}